### PR TITLE
feature2 added - admin role

### PR DESCRIPTION
**Pull request by krzysiekpom**
master + feature1 + added Admin role (boolean value in User model)

### CODE REVIEW

**Comment by socjopata** [comment]
Like I said via e-mail - when proceeding with the pull requests, you have to make branches out of the feature branches and point them appropriately while making pull request. It this case you created a PR pointing to master. This way we got a PR with two commits, where one of it is included in the previous PR. Making the review process a pain to do and getting yourself into merge issues later. I'll wait with the review until you fix this.

**Comment by krzysiekpom** [comment]
Base branch changed to feature1

File: db/seeds.rb [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+}
+
+#create admin role
+User.create!(
+  email: "test@mail.pl",
+  gender: "M",
+  age: 30,
+  admin: true,
+  password: "secret"
+)
```
Comment (socjopata) [inline db/seeds.rb]: Empty line

File: spec/factories.rb [hunk 2]
```diff
@@ -4,5 +4,10 @@
     gender           "M"
     age              42
     password         "secret"
+    admin            false
+
+    factory :admin do
+      admin            true
+    end
   end
 end
```
Comment (socjopata) [inline spec/factories.rb]: Empty line

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
-# RSpec.describe User, type: :model do
-  
-# end
+RSpec.describe User, type: :model do
+  context "has an admin role" do
```
Comment (socjopata) [inline spec/models/user_spec.rb]: Please remind me to talk about specs and their value a bit, after you finish your project. Kudos for making your own research and approaching the topic by yourself, before finishing the project..

File: app/assets/stylesheets/custom.scss [hunk 4]
```diff
@@ -63,6 +63,13 @@ div#main{
   }
 }
 
+input{
+  margin: 5px 0;
+  padding: 5px;
```
Comment (socjopata) [inline app/assets/stylesheets/custom.scss]: Consider sorting these alphabetically

File: spec/models/user_spec.rb (hunk 3 above)
Comment (krzysiekpom) [reply spec/models/user_spec.rb]: removed the file

**Comment by krzysiekpom** [comment]
In order to not repeat myself (and the changes made in feature1) I merged feature1 into feature2 locally and only then applied the changes. It seems to work well, let me know in case there are any issues with that.

**Comment by socjopata** [comment]
Please revert the merge and rebase feature2 on feature1

**Comment by krzysiekpom** [comment]
Should I do this for other branches as well? I'm guessing this is in order to get a cleaner git history, if so, it would make sense to be consistent I think.

**Comment by socjopata** [comment]
Yes, that's standard procedure for working with feature branches, in this configuration

**Pull request merged**